.github/workflows/commitflow-ui.yml

```diff
@@ -0,0 +1,60 @@
+name: UI CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+    paths:
+      - '**.js'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Caching npm with setup node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
```

Comment:
Question: Lowest we support is 16? Should we also test for other node versions with the matrix?

Curious: Will it have any significant effect though for other node versions if we don't test with them?

Reply:
style listing with version 10 was failing, so currently will be using node-version: 14.